presets.go

```diff
@@ -106,8 +106,11 @@ var PRESETS = map[string]QueryPreset{
 	"iran": QueryPreset{
 		include: []string{"iran", "tehran", "mashhad", "isfahan", "esfahan", "karaj", "shiraz", "tabriz", "qom", "ahvaz", "ahwaz", "kermanshah", "urmia", "rasht", "kerman"},
 	},
-	"congo": QueryPreset{
-		include: []string{"congo", "drc", "kinshasa", "lubumbashi", "bukavu", "kananga", "goma"},
+	"congo Kinshasa": QueryPreset{
+		include: []string{"congo+kinshasa","drc", "cod", "kinshasa", "lubumbashi", "bukavu", "kananga", "goma", "mbuji+mayi", "likasi", "kolwezi","kalemie","uvira","matadi","moba","Kamina","Kabalo","kungurume"},
```

Comment:
```suggestion
	"congo kinshasa": QueryPreset{
		include: []string{"congo+kinshasa","drc", "cod", "kinshasa", "lubumbashi", "bukavu", "kananga", "goma", "mbuji+mayi", "likasi", "kolwezi","kalemie","uvira","matadi","moba","kamina","kabalo","kungurume"},
```
use consistent lowercase.

Reply:
Thank you